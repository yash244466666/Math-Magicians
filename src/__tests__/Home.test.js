/* eslint-disable no-unused-expressions */
import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from '../components/pages/Home';

describe('test for Home Component', () => {
  it('should render home component', () => {
    expect(render(<Home />)).toMatchSnapshot();
  });

  it('should query text component', () => {
    render(<Home />);
    screen.debug();

    expect(screen.queryByText(/Welcome to our page!/)).not.toBeNull;
  });
});
