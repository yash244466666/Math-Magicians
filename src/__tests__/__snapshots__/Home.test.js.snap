// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Home Component should render home component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h3>
          Welcome to our page!
        </h3>
        <p>
          I called the it department about that ransomware because of the old antivirus, but he said that we were using avast 2021 loop back we have to leverage up the messaging, yet incentivize adoption, nor moving the goalposts, yet low-hanging fruit let's put a pin in that. Circle back through the lens of radical candor, nor enough to wash your face high-level. Fast track note for the previous submit: the devil should be on the left shoulder horsehead offer, or pushback, so run it up the flagpole, ping the boss and circle back, nor radical candor. Design thinking old boys club, for re-inventing the wheel high turnaround rate.
        </p>
        <p>
          Shotgun approach identify pain points. Pivot post launch big data creativity requires you to murder your children let's circle back tomorrow. To be inspired is to become creative, innovative and energized Quick-win. I know you're busy my supervisor didn't like the latest revision you gave me can you switch back to the first revision?, so pipeline, or the horse is out of the barn thinking outside the box 60% to 30% is a lot of persent.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h3>
        Welcome to our page!
      </h3>
      <p>
        I called the it department about that ransomware because of the old antivirus, but he said that we were using avast 2021 loop back we have to leverage up the messaging, yet incentivize adoption, nor moving the goalposts, yet low-hanging fruit let's put a pin in that. Circle back through the lens of radical candor, nor enough to wash your face high-level. Fast track note for the previous submit: the devil should be on the left shoulder horsehead offer, or pushback, so run it up the flagpole, ping the boss and circle back, nor radical candor. Design thinking old boys club, for re-inventing the wheel high turnaround rate.
      </p>
      <p>
        Shotgun approach identify pain points. Pivot post launch big data creativity requires you to murder your children let's circle back tomorrow. To be inspired is to become creative, innovative and energized Quick-win. I know you're busy my supervisor didn't like the latest revision you gave me can you switch back to the first revision?, so pipeline, or the horse is out of the barn thinking outside the box 60% to 30% is a lot of persent.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
